### ISSUE TITLE
Does not work on GitHub mobile site.

### ISSUE THREAD

**Issue by Grossdm** [musically-ut/lovely-forks#46]
To have Lovely Forks display its magic when viewing GitHub with Firefox Android, I must either select "Desktop version" (if GitHub offers this on a given page) or ask Fennec to "Request desktop site" in its menu.

I don't know if the GitHub mobile site would allow Lovely Forks to work at all or with reasonable modifications to LF's code.

GitHub's mobile site unfortunately hides various useful site features.

Thanks,
Doug

**Comment by musically-ut** [comment]
Good point; thanks for bringing it to my notice. I'll need to explore what the HTML structure of the mobile site is.

Also, I'm not very happy with all the info/features GitHub's mobile site hides either.

**Comment by Grossdm** [comment]
I've begun switching to the Desktop version of GitHub before writing any posts.
From what I can tell, that the only way to have the preview option.

Hmm... Could we blame this on Microsoft now? 😉

**Comment by Grossdm** [comment]
@musically-ut, I don't want to "talk down" to you (again, I'm not a dev) but are you familiar with the "Responsive Design Mode" in desktop browsers?  
  
I suspect that this would be the first tool to try to analyze the trouble that GitHub's mobile site causes with Lovely Forks.
Both Chrome 😝 and Firefox 👍 have RDM in their Developer Tools (even in the release versions). 
RDM allows you to simulate various mobile devices.
You can even add a custom device (at least with Firefox). 
  
 Just hoping to advance this issue a bit. 😁

Thanks again for LF.

**Comment by musically-ut** [comment]
I'm afraid that technology is not the limiting factor when it comes to this issue; the limiting factor is my time. I've been kept a little too busy with other things. 

However, I will get back to it after this upcoming week; thanks for the bump. :)
